elasticsearch_service/helpers/consumer.go

```diff
@@ -0,0 +1,41 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"github.com/segmentio/kafka-go"
+	"log"
+	"strings"
+)
+
+type KafkaConsumer struct {
+	Reader   *kafka.Reader
+	EsClient ElasticsearchClient
+}
+
+func (consumer *KafkaConsumer) Consume() {
+	//go func() {
+	fmt.Println("KafkaConsumer is ready to consume new messages.")
+	for {
+
+		m, err := consumer.Reader.ReadMessage(context.Background())
+		if err != nil {
+			break
+		}
+
+		fmt.Printf("message at offset %d: %s = %s\n", m.Offset, string(m.Key), string(m.Value))
+
+		s := strings.Split(string(m.Value), " ")
+
+		if s[0] == "CREATED" || s[0] == "UPDATED" {
```

Comment:
Length check is needed here, or IndexOutOfBounds may be thrown.

Reply:
Done.